pynuodb/session.py

```diff
@@ -139,6 +139,22 @@ def __init__(self, host, port=None, service="Identity", timeout=None,
                 else:
                     raise
 
+    @staticmethod
+    def _extract_options(options):
+        expected_extracted_options = ['password', 'user',
+                             'trustStore', 'verifyHostname', 'allowSRPFallback', 'ciphers',
+                             'ipVersion', 'direct']
```

Comment:
Do we have a reference implementation of the driver that we base changes like this on? What do JDBC or the C++ driver?

Reply:
I copy pasted that to this review as a comment